package.json

```diff
@@ -22,8 +22,10 @@
     "@react-native-community/masked-view": "^0.1.10",
     "@react-navigation/native": "^5.9.0",
     "@react-navigation/stack": "^5.13.0",
+    "@storybook/react-native": "^5.3.25",
```

Comment:
Are you sure you want to use the `^` for your dependency version?

Reply:
Sorry, I missed it when reviewing
I fixes this